test/Solana.SvmCpiEventsClient.Integration.test.ts

```diff
@@ -62,6 +75,83 @@ describe("SvmCpiEventsClient (integration)", () => {
     const signature = await deposit(signer, solanaClient, depositInput, decimals);
     return { signature, depositInput };
   };
+  const uint8ArrayToBigNumber = (arr: Uint8Array): BigNumber => {
+    if (arr.length === 0) {
+      return BigNumber.from(0); // or handle this case appropriately
+    }
+
+    const hex = Array.from(arr)
+      .map((byte) => byte.toString(16).padStart(2, "0"))
+      .join("");
+
+    return BigNumber.from("0x" + hex);
+  };
+
+  const sendRequestSlowFill = async () => {
+    const eventAuthority = await getEventAuthority();
+
+    const latestSlot = await solanaClient.rpc.getSlot({ commitment: "confirmed" }).send();
+    const currentTime = await getTimestampForSlot(solanaClient.rpc, Number(latestSlot));
+
+    const relayData: SvmSpokeClient.RequestSlowFillInstructionDataArgs["relayData"] = {
+      depositor: signer.address,
+      recipient: signer.address,
+      exclusiveRelayer: signer.address,
+      inputToken: mint.address,
+      outputToken: getRandomSvmAddress(),
+      inputAmount: tokenAmount,
+      outputAmount: tokenAmount,
+      originChainId: 1,
+      depositId: new Uint8Array(),
+      fillDeadline: Number(currentTime) + 60 * 30, // 30‑minute deadline
```

Comment:
There might be a misunderstanding here - this should be copied 1:1 from the deposit arguments, otherwise the RelayData hash won't match and the slow fill request will be invalid.